packages/zoe/test/swingsetTests/zoe/vat-bob.js

```diff
@@ -95,9 +95,17 @@ const build = async (log, zoe, issuers, payments, installations, timer) => {
         optionValue[0].description === 'exerciseOption',
         details`wrong invitation`,
       );
-      assert(moolaAmountMath.isEqual(optionValue[0].underlyingAsset, moola(3)));
       assert(
-        simoleanAmountMath.isEqual(optionValue[0].strikePrice, simoleans(7)),
+        moolaAmountMath.isEqual(
+          optionValue[0].underlyingAssets.UnderlyingAsset,
```

Comment:
Here is where I figured out that the `coveredCall` could now support multiple assets :D